system_tests/snap_sync_test.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package arbtest
+
+import (
+	"context"
+	"math/big"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/params"
+
+	"github.com/offchainlabs/nitro/arbnode"
+	"github.com/offchainlabs/nitro/arbos/l2pricing"
+	"github.com/offchainlabs/nitro/util"
+)
+
+func TestSnapSync(t *testing.T) {
```

Comment:
problem with the test is it also passes if you set snapsync.Enable to false

I think best way to make sure we really did use snap-sync is to try and load message or messagemetadata a message that's befor the snap sync (say, 7). If it exists - we didn't really snap-sync and the test should fail.
Please make the change and then check the the test does fail if we disable snap_sync in node C's creation.

Reply:
Done.